src/services/item/service.ts

```diff
@@ -682,8 +682,15 @@ export class ItemService {
       await this.hooks.runPreHooks('copy', actor, repositories, { original });
     }
 
-    // TODO: args?
-    const { copyRoot, treeCopyMap } = await itemRepository.copy(item, actor, parentItem);
+    const siblings: Item[] = [];
+    if (parentItem) {
+      siblings.push(...(await itemRepository.getChildren(parentItem)));
+    } else {
+      const itemMemberships = await itemMembershipRepository.getAccessibleItems(actor, {}, null);
```

Comment:
I wonder if we really want to do that in the end.. this call is quite heavy (but it might be heavy on other ends of the call, but getting accessible is not light)

Reply:
We should implement a getAccessibleItemsName which is a light similar function.

We will set a limit of 999 copy, after that, we're having multiple items (1000)